### ISSUE TITLE
Link to VSIX file uses invalid certificate

### ISSUE THREAD

**Issue by SactoJosh** [NuGet/NuGetDocs#342]
The "Installing a CI build" link to the VSIX file on the "Installing NuGet" page has anchor: http://build.nuget.org/NuGet.Tools.vsix  Using Google Chrome, at least, gives an error trying to get this file because the build.nuget.org redirects to https and the certificate that is returned is for nugetbuild.cloudapp.net.

To alleviate this problem, the link on the page could be changed to direct to:
https://nugetbuild.cloudapp.net/NuGet.Tools.vsix

**Comment by maartenba** [comment]
Just tried this in Chrome 43 without a certificate issue. Which version of Chrome are you using?

**Comment by SactoJosh** [comment]
I'm using Chrome 43 as well. Here is a screen shot of the error and
certificate. Let me know if it doesn't come through...

[image: Inline image 1]

**Comment by maartenba** [comment]
Can you try visiting www.nuget.org first and then try again?

**Comment by SactoJosh** [comment]
Visiting www.nuget.org first did the trick. I Was having trouble using the
addon manager in Visual Studio. I was on a machine where I couldn't run VS
as an admin so every time I tried to install NuGet from the addon manager,
it launched hundreds of vsix installer processes slowing my machine to a
crawl. So I Googled around trying to find a direct download for the vsix
file and the first link I came across was
https://docs.nuget.org/consume/installing-nuget so that was the first nuget
site I visited.

Thanks for the assist.


On Fri, Jun 12, 2015 at 12:19 AM, Maarten Balliauw <notifications@github.com
